packages/react/src/hooks/useTrackVolume.ts

```diff
@@ -154,3 +153,51 @@ export function useMultibandTrackVolume(
 
   return frequencyBands;
 }
+
+function calculateLogBinEdges(minFreq: number, maxFreq: number, numBins: number): number[] {
```

Comment:
probably worth looking at Bark Scale  also (https://en.wikipedia.org/wiki/Bark_scale) as it maps psychoacoustics.

Reply:
oh, nice! might be even better!

Reply:
Mel Scale - https://en.wikipedia.org/wiki/Mel_scale may be better.

Reply:
One is pitch and the other is loudness scale.